### PR TITLE
Add Rock Pi 3A compatibility

### PR DESCRIPTION
**Pull request by etrevis**

### CODE REVIEW

File: adafruit_platformdetect/constants/boards.py [hunk 1]
```diff
@@ -165,6 +165,7 @@
 
 RADXA_ZERO = "RADXA_ZERO"
 RADXA_CM3 = "RADXA_CM3"
+RADXA_3A = "ROCK_PI_3A"
```
Comment (makermelissa) [inline adafruit_platformdetect/constants/boards.py:168]: Could we change this to ROCK_PI_3A to match the other ROCK PI boards?

**Review by makermelissa — CHANGES_REQUESTED** [review]
Just for the purpose of trying to keep a large number of boards organized, I'd like to stick with consistent naming. Otherwise, the PR looks good.

File: adafruit_platformdetect/constants/boards.py (hunk 1 above)
Comment (etrevis) [reply adafruit_platformdetect/constants/boards.py:168]: Ok, will change it also downstream (blinka)

**Review by makermelissa — APPROVED** [review]
Thank you. Look great.

**Pull request merged**